azure/azure-pr-teardown-pipeline.yml

```
@@ -23,13 +23,15 @@ jobs:
       vmImage: 'ubuntu-latest'
     steps:
       - checkout: self
+        path: "s/$(SERVICE_NAME)"
 
       - bash: |
           echo $(action_pr_number)
           WORKSPACE="pr-"$(ACTION_PR_NUMBER)
           echo $WORKSPACE
           echo "##vso[task.setvariable variable=WORKSPACE]$WORKSPACE"
           echo "##vso[task.setvariable variable=ACTION_PR_NUMBER]$(ACTION_PR_NUMBER)"
+          echo $(SERVICE_NAME)
```

Very minor, but any reason to wrap. Could be the same as line 31 with WORKSPACE for consistency?
```suggestion
          echo $SERVICE_NAME
```

---

Can probably ignore this. As per Rob's comment looks like we'll be removing it.